docs/team/junxiang.md

```diff
@@ -9,28 +9,59 @@ NotUS is a quick and simple, Command Line Interface (CLI) based, note-taking app
 *Roles (short for ‘in charge of role’): Description*
 
 - Co-team lead: Responsible for overall project coordination alongside Ramana.
-- Code quality: Looks after code quality, ensures adherence to coding standards, etc.
-- Documentation: Responsible for the quality of various project documents.
+- Software Architect: Designed the software  architecture of the programme with Nazryl. Ensures the logic of the code structure. 
 - Integration: Worked with Ramana for the versioning of the code, maintaining the code repository, integrating various parts of the software to create a whole.
-- In charge of `list-n`,`archive-n`, `unarchive-n`, `find-n`, of the code:
 
 ## Summary of Contributions
 
 ### Features implemented
-1. Set up the skeleton code used in the project, including the following classes:
-    * `Command` Classes
-    * `Formatter` Classes
-    * Test Case Classes
+- Implemented `TaggableObject` class which is inherited by `Note` and `Event`. 
+    - Taggable objects can have multiple tags and different Taggable objects can share same tag(s). They can also be filtered by their tags.
+- Implemented `Tag` class which contains the name and of the tag and its color.
+    - Added color support for user to better differentiate the tags.
+- Implemented `TagManager` class which manages all the tags in application. 
+    - A hashmap is used where each unique tag are the keys and the arraylist of taggable objects that share the same tag are the mapped values.
+- Implemented the functionality to create tag, delete tag, as well as tag or untag taggable objects.
+- Implemented `ParserManager` class which manages the creation of different parser functions to parse the user input to the required parameters.
+- Implemented `SystemException` class which extends from `Exception` class.
+    - Within it, there is an enumeration for all possible types of exception that may occur and each enum value contains a unique string as its message.
+- Implemented `Formatter` class which formats the different objects, such as Note, Event and String with a specified layout into a String to be printed.
     
 ### Code contributed
+[RepoSense link](https://nus-cs2113-ay2021s1.github.io/tp-dashboard/#breakdown=true&search=chongjx&sort=groupTitle&sortWithin=title&since=2020-09-27&timeframe=commit&mergegroup=&groupSelect=groupByRepos&checkedFileTypes=docs~functional-code~test-code~other)
+
+### Project management
+- Managed releases `v1.0` and `v2.0` on GitHub
```

Comment:
Could add v2.1 since you are most probably gonna do it anyways

Reply:
Noted.